docs/configuration.md

```diff
@@ -58,7 +58,7 @@ A comma separated list of prisons for which PVB2 API will return slots along wit
 are available at the requested prison.
 
 
-### `NOMIS_STAFF_OFFENDER_RESTRICTIONS_ENABLED`
+### `NOMIS_STAFF_RESTRICTIONS_ENABLED`
```

Comment:
Is the use of 'staff' or potentially redundant or misleading?  The restrictions relate to an offender or visitor  ... Perhaps we no longer need to refer to 'staff'

Reply:
It's convinient to have STAFF in env var names as some of the env vars are to turn on flags for the public site

Reply:
Cool